src/rez/cli/test.py

```diff
@@ -14,11 +25,17 @@ def setup_parser(parser, completions=False):
     parser.add_argument(
         "--nl", "--no-local", dest="no_local", action="store_true",
         help="don't load local packages")
+    parser.add_argument(
+        "--interactive", dest="interactive", default=None,
+        help="resolves an interactive environment for running the tests.")
+    parser.add_argument(
+        "--variant", dest="variant", default=0, type=int,
+        help="variant number which should be used to set the interactive env. Works only with --interactive option")
```

Comment:
This should work in the general case also, it makes sense to specify a variant to run tests on.